/**:     # Node namespace
  updated_turtlebot3_node:
    ros__parameters:

      opencr:
        id: 200
        baud_rate: 1000000
        protocol_version: 2.0

      wheels:
        separation: 0.160
        radius: 0.033

      motors:
        profile_acceleration_constant: 214.577

        profile_acceleration: 0.0

      sensors:
        bumper_1: 0
        bumper_2: 0
        illumination: 0
        ir: 0
        sonar: 0

/**:     # Node namespace
  diff_drive_controller:
    ros__parameters:

      odometry:
        publish_tf: true
        use_imu: true
        frame_id: "world"
        child_frame_id: "default_ns/base_footprint"
        initial_transform:
          x: 0.0
          y: 0.0
          yaw: 0.0
